.github/workflows/generate_library.yml

```diff
@@ -59,11 +59,10 @@ jobs:
         poetry lock --no-update
       working-directory: ansys-grantami-bomanalytics-openapi
 
-    - name: "Install and run black"
+    - name: "Install and run pre-commit"
       run: |
-        pip install black
-        black ./src ./tests
-      working-directory: ansys-grantami-bomanalytics-openapi
+        pip install pre-commit
+        ! pre-commit run --all-files
```

Comment:
It might be a stupid question, what does the "!" do?

Reply:
ah I get it. I suppose this is ok, seeing as we know the hooks are never going to succeed